core/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class UnimplementedUCException(Exception):
+    """Returned when an uncaught exception is raised"""
+
+    def __init__(self):
+        self.failure_message = "Unknown validation error, Please contact support"
+
+
+def unimplemented_uc_error_handler(error: UnimplementedUCException):
+    return {
+        "detail": "Uncaught workbook validation failure",
+        "validation_errors": error.failure_message,
+        "status": 500,
+        "title": "Bad Request",
+    }, 500
```

Comment:
Don't think we need `validation_errors` in this case, so we can remove that and the `__init__` from the exception I think. We can just handle a 500 error from the status code and then provide some static content.